backend/app/Services/Connectivity/ActionExecutor.php

```
@@ -160,6 +166,82 @@ private function updateWorkOrderQty(array $params, array $data, mixed $fieldValu
+            'increment' => $increment,
+            'step' => $stepResult,
+            'counted_to_work_order' => $countedToWorkOrder,
+        ];
+    }
+
+    /**
+     * The line's current running work order (most recent IN_PROGRESS).
+     */
+    private function activeWorkOrderOnLine(int $lineId): ?WorkOrder
+    {
+        return WorkOrder::where('line_id', $lineId)
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\WorkOrder::where().


(staticMethod.notFound)